[DOC] Update TraceQL query editor doc (#91910)
grafana/grafana: docs/sources/datasources/tempo/service-graph.md

You use the Service Graph to detect performance issues; track increases in error, fault, or throttle rates in services; and investigate root causes by viewing corresponding traces.

> {{< figure src="/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-node-graph.png" class="docs-image--no-shadow" max-width="500px" caption="Screenshot of a Node Graph" >}}

Reviewer: Alt text is required, but captions aren't, so swapping this.
```suggestion
{{< figure src="/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-node-graph.png" class="docs-image--no-shadow" max-width="500px" alt="Screenshot of a Node Graph" >}}
```

Service graph view displays a table of request rate, error rate, and duration metrics (RED) calculated from your incoming spans. It also includes a node graph view built from your spans.

> {{< figure src="/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png" class="docs-image--no-shadow" max-width="500px" caption="Screenshot of the Service Graph view" >}}

Reviewer: ```suggestion
{{< figure src="/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png" class="docs-image--no-shadow" max-width="500px" alt="Screenshot of the Service Graph view" >}}
```

> For details, refer to the [Service Graph view documentation](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png).

Reviewer: Assuming this should be a link
```suggestion
For details, refer to the [Service Graph view documentation](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png).
```

Author: I updated the URL to the correct one instead of the image. Good catch!

> For details, refer to the [Service Graph view documentation](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png).

Author: Yes, it should be a link. Your suggestion has the same mistake that I had with the image. I've updated the link here. 
```suggestion
For details, refer to the [Service Graph view documentation](/docs/tempo/<TEMPO_VERSION>/metrics-generator/service-graph-view/).
```
